docs/spec/manifest-v2-2.md

```diff
@@ -0,0 +1,255 @@
+# Image Manifest Version 2, Schema 2
+
+This document outlines the format of of the V2 image manifest, schema version 2.
+The original (and provisional) image manifest for V2 (schema 1), was introduced
+in the Docker daemon in the [v1.3.0
+release](https://github.com/docker/docker/commit/9f482a66ab37ec396ac61ed0c00d59122ac07453)
+and is specified in the [schema 1 manifest definition](./manifest-v2-1.md)
+
+This second schema version has two primary goals. The first is to allow
+multi-architecture images, through a "fat manifest" which references image
```

Comment:
@aaronlehmann We're discussing this PR with @coolljt0725 and we were thinking about `docker build`. How would `docker build` create a multi-arch image? If not on build, then on push (that's a detail). Now if we say "it only creates arch-specific manifests", then we should have a way of creating a arch-independent manifest (sort of a meta-manifest) that references the arch-specific manifests. If that makes  sense, that could be an argument against fat manifests. WDYT?